Formula/sdl12-compat.rb

```diff
@@ -0,0 +1,31 @@
+class Sdl12Compat < Formula
+  desc "SDL 1.2 compatibility layer that uses SDL 2.0 behind the scenes"
+  homepage "https://github.com/libsdl-org/sdl12-compat"
+  url "https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz"
+  sha256 "5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014"
+  license all_of: ["Zlib", "MIT-0"]
+  head "https://github.com/libsdl-org/sdl12-compat.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "sdl2"
+
+  conflicts_with "sdl", because: "sdl12-compat is a drop-in replacement for sdl"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+           "-DSDL2_PATH=#{Formula["sdl2"].prefix}",
+           "-DSDL12DEVEL=1", "-DSDL12TESTS=0"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    ln_s lib/"pkgconfig/sdl12_compat.pc", lib/"pkgconfig/sdl.pc"
+  end
+
+  test do
+    on_macos do
+      assert_predicate lib/"libSDL.dylib", :exist?
+      assert_predicate lib/"libSDL-1.2.0.dylib", :exist?
+    end
```

Comment:
```suggestion
    assert_predicate lib/shared_library("libSDL"), :exist?
    assert_predicate lib/shared_library("libSDL-1.2.0"), :exist?
```
This might work

Reply:
I'll try it!